apps/meteor/app/lib/server/functions/saveCustomFieldsWithoutValidation.ts

```diff
@@ -12,7 +15,14 @@ const getCustomFieldsMeta = function (customFieldsMeta: string) {
 		throw new Meteor.Error('error-invalid-customfield-json', 'Invalid JSON for Custom Fields');
 	}
 };
-export const saveCustomFieldsWithoutValidation = async function (userId: string, formData: Record<string, any>): Promise<void> {
+export const saveCustomFieldsWithoutValidation = async function (
+	userId: string,
+	formData: Record<string, any>,
+	options?: {
+		_updater?: Updater<IUser>;
+		session?: ClientSession;
+	},
```

Comment:
To me, `options` is T | undefined, but if `options` is provided, the keys inside it shouldn't be undefined 🤔 

That would remove the need for the `if (!updater)` inside the func

Reply:
`if (!updater)` is used in order to "remove" the responsibility of the function to commit the updater. If you pass _updater as option, you have to commit the updater manually. Otherwise the function works as it did before.

Both parameters are independent and optional. You can pass just the updater, or just the session, and they should work despite the other being present or not.